Engine/ResourceManagement/Importer/ModelImporters/MeshImporter.cpp

```diff
@@ -24,6 +24,8 @@ FileData MeshImporter::ExtractMeshFromAssimp(const aiMesh* mesh, const aiMatrix4
 
 	node_transformation = scaling_matrix * node_transformation;
 
+	float3x3 node_rotation = Utils::GetTransform(mesh_current_transformation, 1.0f).RotatePart();
```

Comment:
This probably will have to change with the new importing stuff

Reply:
Yes I think you can just remove it when merging